docs/dev/bug_fixes.md

```diff
@@ -1,13 +1,13 @@
 # Bug Fixes
```

Comment:
I think we can just delete this document as it's not being maintained.

It would be too much effort to log every PR in here.